appveyor.yml

```diff
@@ -0,0 +1 @@
+# Appveyor is only enabled on modules that support windows
```

Comment:
Having a broken appveyor file will cause issues when trying to enable appveyor as the "Only run on branches with appveyor.yml" setting will not trigger.